Partition_2/include/CGAL/Partition_2/Rotation_tree_2.h

```diff
@@ -70,24 +70,33 @@ class Rotation_tree_2 : public internal::vector< Rotation_tree_node_2<Traits_> >
       }
    };
 
+   struct Equal {
```

Comment:
I'm surprized this is needed

Reply:
It is. For `std::unique()`.

Reply:
The doc of `std::unique` says that operator== is used so it should compile without it.

Reply:
It doesn't. Also before we derived from Point, now it is a data member.  And we want the data member to be unique.